automates/text_reading/src/test/scala/org/clulab/aske/automates/TestUtils.scala

```diff
@@ -145,6 +160,43 @@ object TestUtils {
       arg2Strings.foreach(arg2String => identifierDescriptionPairs should contain ((arg1String, arg2String)))
     }
 
+    def testUnaryEvent(mentions: Seq[Mention], eventType: String, arg1Role: String, desired: Seq[String]): Unit = {
```

Comment:
Is this test actually used anywhere? If not, we can delete this.

Reply:
Yes, that is for the function fragment tests!